docker.go

```diff
@@ -98,6 +161,22 @@ func (container *Container) EmptyVolumes() *Container {
 	return container
 }
 
+// SetPersistentVolume defines persistent properties for volume
+//      size: size of the volume in MiB
+func (v *Volume) SetPersistentVolume(size int) *PersistentVolume {
```

Comment:
Again, what if you want to update an existing persistent value but leave the size untouched? AFAICS, in this case you wouldn't want to set it.